portality/scripts/admin_journalcsv.py

```
@@ -12,12 +12,12 @@
 
     parser = argparse.ArgumentParser()
     parser.add_argument("-o", "--out", help="output file for admin csv")
-    parser.add_argument("-a", "--obscure_accounts", help="true if accounts should be obscured", default="true")
+    parser.add_argument("-a", "--do_not_obscure_accs", help="set this flag if the accounts should not be obscured", action='store_false')
     args = parser.parse_args()
 
     # ~~-> Journal:Service~~
     journalService = DOAJ.journalService()
-    journalService.admin_csv(file_path=args.out, obscure_accounts=args.obscure_accounts)
+    journalService.admin_csv(file_path=args.out, obscure_accounts=args.do_not_obscure_accs)
```

I like the default to obscured approach, very nice.